src/source/PeerConnection/SessionDescription.c

```diff
@@ -316,6 +326,64 @@ PCHAR fmtpForPayloadType(UINT64 payloadType, PSessionDescription pSessionDescrip
     return NULL;
 }
 
+/*
+ * Extracts a (hex) value after the provided prefix string. Returns true if
+ * successful.
+ */
+static BOOL readHexValue(const PCHAR input, const PCHAR prefix, PINT32 value) {
+  const PCHAR substr = STRSTR(input, prefix);
+  if (substr != NULL) {
+    if (sscanf(substr + STRLEN(prefix), "%x", value) == 1) {
```

Comment:
Combine the two ifs like if (a && b)

Most importantly, we currently don't have a macro for sscanf. Not recommending to use sscanf either. In this case, recommend using 

CHAR valueStr[16];

if (substr != NULL && STATUS_SUCCEEDED(ULTOSTR(value,  valueStr, ARRAY_SIZE(valueStr), 16, NULL)) {
    return TRUE;
} else {
    return FALSE;
}

Reply:
Hmm, this is a bit tricky... my preference would be to use `sscanf` from the standard library, as it can read arbitrary length case-agnostic hex values, which is what we want here, I think? If this sounds okay, I'll put up a PR to get a macro wrapper into the pic code. :-)

Reply:
sscanf will not do much better in terms of handling hex as in your case you are not even trying uint64 but rather int. Sscanf is multiple orders of magnitude in terms of runtime costs. However, this usage is not in a main "hot" path so I am OK perf-wise. As a rule of thumb, the usage of these APIs should be super limited to not proliferate across the SDK to keep the runtime footprint low.